### ISSUE TITLE
Would it be possible to also update the System Software packages?

### ISSUE THREAD

**Issue by OuterWinnie** [0xD34D/FlatpakUpdater#5]
I'm talking about this ones:

![photo_2022-11-15_00-38-37](https://user-images.githubusercontent.com/6392320/201790523-bd5dd001-cc77-4c07-8368-e41b4c6709a4.jpg)

**Comment by 0xD34D** [comment]
Yup.  Totally possible and should be resolved with https://github.com/0xD34D/FlatpakUpdater/commit/13adcb2840694584a242d453a7f332c44c0323ae

**Comment by outerwinnie** [comment]
Oh cool, thanks!
